File: sample-videochat-webrtc/sample-videochat-webrtc/Components/Settings/SettingsSectionModel.m
//
//  SettingsSectionModel.m
//  sample-videochat-webrtc
//
//  Created by Injoit on 2/25/19.
//  Copyright © 2019 Quickblox. All rights reserved.
//

#import "SettingsSectionModel.h"

@interface SettingsSectionModel()

@property (copy, nonatomic) NSString *title;
//@property (strong, nonatomic) NSArray *items;

@end

@implementation SettingsSectionModel

+ (instancetype)sectionWithTitle:(NSString *)title items:(NSArray *)items {
    
    SettingsSectionModel *section = [SettingsSectionModel new];
    section.title = title;
    section.items = items;
    
    return section;
}

@end

File: sample-videochat-webrtc-swift/sample-videochat-webrtc-swift/Components/Settings/SettingsSectionModel.swift
//
//  SettingsSectionModel.swift
//  sample-videochat-webrtc-swift
//
//  Created by Injoit on 12/11/18.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import Foundation

class SettingsSectionModel {
    // MARK: - Properties
    var title = ""
    var items = [BaseItemModel]()
    
    // MARK: - Public Methods
    class func section(withTitle title: String, items: [BaseItemModel]) -> SettingsSectionModel {
        let section = SettingsSectionModel()
        section.title = title
        section.items = items
        return section
    }
}
